src/apollo/queries.js

```diff
@@ -840,7 +840,7 @@ export const MINING_POSITIONS = (account) => {
 export const PAIRS_BULK = gql`
   ${PairFields}
   query pairs($allPairs: [Bytes]!) {
-    pairs(where: { id_in: $allPairs }, orderBy: trackedReserveNativeCurrency, orderDirection: desc) {
+    pairs(first: 200, where: { id_in: $allPairs }, orderBy: trackedReserveNativeCurrency, orderDirection: desc) {
```

Comment:
```suggestion
    pairs(first: 200, where: { id_in: $allPairs }, orderBy: trackedReserveNativeCurrency, orderDirection: desc) {
```
What is this value `trackedReserveNativeCurrency` can we order by liquidity?

Reply:
We've added the `GNO` token to the whitelist on the subgraph https://github.com/dxgraphs/swapr-subgraph/pull/13, so this will be fixed when we'll deploy the changes on the subgraph